apps/web/src/components/FormImageCard.tsx

```diff
@@ -0,0 +1,123 @@
+import {
```

Comment:
Let's remove some unused imports:
  Avatar,
  AvatarGroup,
  Tooltip,
  flexbox,
  SignatureEntity

Reply:
^running the linting command should point out any others too